app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -13,12 +13,18 @@ publishers=Publishers
 payments=Payments
 paymentsWelcomeTitle=Welcome to Brave Payments!
 paymentsWelcomeText1=Brave has created a simple way to for you to contribute to the sites you use most.
-paymentsWelcomeText3=Brave Payments is currently in Beta. With your help, we can fine tune the system over the next several weeks before it rolls out to our release channel.
-paymentsWelcomeText2=Your funds are transferred to the site owner through an anonymous ledger system, which makes it impossible for you ever to be identified based on the sites you visit and support. This allows publishers to stay in business even though you may have blocked their advertisements with Brave. All of this works while keeping your browsing history private.
-paymentsWelcomeTextBold=To begin using Brave Payments, simply flip the switch at the top of this window. The rest is easy.
-paymentsWelcomeText4=Learn more...
-publisherPaymentsTitle=Brave Payments Beta
-paymentsFooterText=Transaction IP addresses anonymized using
+paymentsWelcomeText2=To begin using Brave Payments, simply flip the switch at the top of this window. The rest is easy.
+paymentsWelcomeText3=Brave Payments allows publishers to stay in business even though you may have blocked their advertisements with Brave. All of this works while keeping your browsing history private. Your funds are transferred to the site owner through an anonymous ledger sytem, which makes it impossible for you ever to be identified based on the sites you visit and support.
+paymentsWelcomeText4=Brave Payments is currently in Beta. With your help and feedback, we can fine tune the system through the beta period.
+paymentsWelcomeText5=Need more info?
+paymentsWelcomeText6=for Brave Payments...
+paymentsWelcomeLink=View the FAQ
+publisherPaymentsTitle=Brave Payments
+publisherPaymentsTitleBeta=beta
+paymentsSidebarText1=Our Partners
+paymentsSidebarText2=All transactions are anonymized with technology from:
```

Comment:
@bradleyrichter seems like we should be more clear here. `anonymize` can mean a lot of things. all that we use PIA for is hiding real IP addresses. they do not anonymize payments, etc.